components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -17,6 +17,8 @@ import (
 const (
 	destinationArg        = "--destination"
 	functionContainerName = "lambda"
+	//TODO refactor strings with workspace path
```

Comment:
refactor will be applied in https://github.com/kyma-project/kyma/pull/9111/files